### DISCUSSION TITLE
How soon will the ssd die with new recording mode?

### DISCUSSION THREAD

**Discussion by NormKOALA** [blakeblackshear/frigate#1945] [General]
Is there a way to turn off such an endless circle of write-remove such small videos, And record, as before, to disk only clips with detected persons?

**Comment by jasonpstokes** [comment]
There is a feature request already logged for this: #1885

**Comment by blakeblackshear** [comment]
Hard to say. I have had 24/7 recordings on my ssd for 5 years so far.
